tests/test_integrations/test_palo_alto_scm.py

```
@@ -0,0 +1,185 @@
+    assert creds.envs["PALO_ALTO_SCM_SCOPE"] == "tsg_id:1234567890"
+    assert creds.envs["PALO_ALTO_SCM_HOST"] == "api.strata.paloaltonetworks.com"
+    assert creds.envs["PALO_ALTO_SCM_PROTOCOL"] == "https"
+
+
+def test_palo_alto_scm_client_request_injects_bearer():
+    client = PaloAltoSCMClient(
+        protocol="https",
+        host="api.strata.paloaltonetworks.com",
+        auth_url="https://auth.apps.paloaltonetworks.com/oauth2/access_token",
+        client_id="cid",
+        client_secret="csecret",
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "client_secret"

(S106)